charts/kyverno/values.yaml

```diff
@@ -48,7 +48,13 @@ podAnnotations: {}
 
 podSecurityContext: {}
 
-affinity: {}
+# By default this will make sure two pods don't end up on the same node
+# Changing this to a region would allow you to spread pods across regions
+antiAffinityTopologyKey: "kubernetes.io/hostname"
```

Comment:
How does it work without change `charts/kyverno/templates/deployment.yaml`? Have you tested it locally?